chain/client/src/stateless_validation/partial_witness_tracker.rs

```diff
@@ -129,16 +148,25 @@ impl PartialEncodedStateWitnessTracker {
         &mut self,
         partial_witness: PartialEncodedStateWitness,
     ) -> Result<(), Error> {
-        let rs = self.rs_map.entry(partial_witness.num_parts());
+        // The expected number of parts for the Reed Solomon encoding is the number of chunk validators.
+        let num_parts = self
```

Comment:
We don't need to transmit the num_parts in the partial witness as we can deduce it from the chunk validators.

Reply:
nit. it will be even cleaner to extract this to a function like get_num_parts to make it clear that it is from chunk producers size.